.vscode/tasks.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
add .vscode to .gitignore

Reply:
Including the .vscode folder in the project might have actually some benefits, provided that vscode is used.
In general, this folder can contain useful configuration preferences like, for example, tasks which help starting the development server by using simple keyboard shortcuts when in vscode.
Also to my surprise, these files have been generated automatically when creating the new Angular 14 application from the angular CLI, suggesting that it will become standard from now on.

Reply:
ok :+1: